game/src/constants.js

```diff
@@ -6,5 +6,11 @@ const greetings = [
     'It\'s a plane!',
     '1000% more Cube!',
     'A Whole New Dimension!',
+    'Explosive pleasure!',
+    'A world of unending flight',
+    'come take a flight',
```

Comment:
Can you capitalize the first letter of each greeting?

EX:

```suggestion
    'Come take a flight',
```

Reply:
Sure